cmake/GNDStk_dependencies.cmake

```diff
@@ -43,5 +51,5 @@ FetchContent_MakeAvailable(
     catch-adapter
     Log
     pugixml-adapter
-    json-adapter
+    json
```

Comment:
Given the `Populate` block above, you don't need to list `json` in this block.  `MakeAvailable` is shorthand for the canonical pattern for checking if populated and then adding the subdirectory.